examples/modular_commands/commandset_complex.py

```diff
@@ -42,7 +41,7 @@ def do_durian(self, args: List[str]):
         self._cmd.poutput(', '.join(['{}']*len(args)).format(*args))
 
     def complete_durian(self, text: str, line: str, begidx: int, endidx: int) -> List[str]:
-        return utils.basic_complete(text, line, begidx, endidx, ['stinks', 'smells', 'disgusting'])
+        return self._cmd.basic_complete(text, line, begidx, endidx, ['stinks', 'smells', 'disgusting'])
```

Comment:
It feels awkward to be accessing a protected member in example code.

Reply:
`_cmd` is a protected member of `CommandSet`. If we want to change it to public, then it should first be changed in 1.3 and merged into 2.0.

Reply:
@anselor What are your thoughts of making `CommandSet._cmd` public?

Reply:
In general, protected members imply they're intended to be accessible by subclasses, while accessing it external to the class is discouraged. This is an example which extends CommandSet. The variable will be set or be None depending on whether the CommandSet is registered. Wasn't really intending it to be a public way to access to the cmd2 app.

If you really think they should be public, I guess that's fine. It'll just be None sometimes, depending on the state.

Reply:
I'm OK leaving it protected since it really is only meant to be used as @anselor described.